charts/renku/requirements.lock

```diff
@@ -24,4 +24,4 @@ dependencies:
   repository: https://kubernetes-charts.storage.googleapis.com
   version: 1.6.0
 digest: sha256:a71bb5d25af250ea5280765b3a433cfeb63dd077ca58ae0816ad07e5bd2157c7
-generated: "2019-05-27T15:27:30.285351+02:00"
+generated: "2019-06-07T00:51:36.835977+02:00"
```

Comment:
I think this doesn't need to be committed.

Reply:
removed